src/InformationProcessing.go

```diff
@@ -99,10 +105,19 @@ func getFullSkiString(ids []string) []string {
 func getUserAndHost(connectionURL string) (string, string) {
 	var tokens []string
 	idx := strings.IndexRune(connectionURL, ':')
+	if !strings.Contains(connectionURL, "@") {
```

Comment:
nope, 
idx checks for a colon, e.g. if it's a dbid:user@host type connection or a simple user@host type connection.
What I'm checking for, is the @ itself, meaning if the provided address is valid, or if it's something like "unknown"